pkg/skaffold/kubernetes/log.go

```diff
@@ -193,7 +197,10 @@ func (a *LogAggregator) streamRequest(ctx context.Context, headerColor color.Col
 			// Read up to newline
 			line, err := r.ReadString('\n')
 			if err == io.EOF {
-				a.printLogLine(headerColor, prefix, "<Container was Terminated>\n")
+				// Unless the context was interrupted, this means that the container was stopped.
+				if ctx.Err() != context.Canceled {
```

Comment:
another time I see this happen is when I run `skaffold dev` after `skaffold run` - the "tail end" of the container logs gets picked up.